build/pkgs/giac/spkg-configure.m4

```diff
@@ -5,7 +5,7 @@ SAGE_SPKG_CONFIGURE([giac], [
        m4_pushdef([GIAC_MAX_VERSION], [1.9.999])
        AC_CACHE_CHECK([for giac >= ]GIAC_MIN_VERSION[, <= ]GIAC_MAX_VERSION, [ac_cv_path_GIAC], [
          AC_PATH_PROGS_FEATURE_CHECK([GIAC], [giac], [
-            giac_version=$($ac_path_GIAC --version 2> /dev/null | tail -1)
+            giac_version=$($ac_path_GIAC --version 2> /dev/null | tail -n -1)
```

Comment:
Here's another `-n -1` that should be `-n 1`

Reply:
ok